x-pack/plugins/monitoring/server/alerts/thread_pool_rejections_alert_base.ts

```diff
@@ -178,10 +179,10 @@ export class ThreadPoolRejectionsAlertBase extends BaseAlert {
       ],
     };
   }
-
   protected executeActions(
     instance: AlertInstance,
-    alertStates: AlertThreadPoolRejectionsState[],
+    { alertStates }: { alertStates: AlertState[] },
+    item: AlertData | null,
```

Comment:
This was getting the wrong data was not able to destructure the values further down resulting in a bug for Threadpool alerts.